maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -601,6 +601,18 @@ class AndroidDriver(
         }
     }
 
+    override fun setOrientation(orientation: DeviceOrientation) {
+        // Disable accelerometer based rotation before overriding orientation
+        dadb.shell("settings put system accelerometer_rotation 0")
```

Comment:
Random thought - not sure it's something we should care about - but from my understanding the "auto-rotate" that is disabled here is never re-enabled. 

That also seems to be true for the `user_rotation` system settings: the last `setOrientation` call will "persist" the chosen `user_rotation` system setting beyond the current running flow.

If someone is reusing the same android device to run multiple flows one after the other, it might introduce some side effect between them: if one flow is using `setOrientation` commands while another expect the orientation to be in "auto-rotate" mode (or expect a "default" orientation).

Reply:
Hey @vbarthel-fr 👋 first off, thanks so much pulling and performing an android test for me!

You bring up a good topic here that I hadn't thought about yet. (And thanks for the pointer to `resetProxy` as an example being called from `close()`.)

Do you foresee any holes in doing something like this:
1. The first time that `setOrientation` is called for a flow we store the current values of `accelerometer_rotation` and `user_rotation` (and for iOS just the `XCUIDevice.shared.orientation`) 
2. From the `close()`, if those were saved we reset them to the original values.

Reply:
Yep, that sounds good to me :)

But I'm not knowledgeable enough to know for sure that is OK to rely on `AndroidDriver.close()`. Tomorrow, I will try to have a closer look at the source code, starting by understanding when `close()` is called.